devtools/gettool/getprotoc.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"archive/zip"
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+func getProtoC(version, output string) error {
+
+	var variant string
+	binFile := "bin/protoc"
+	switch runtime.GOOS + "-" + runtime.GOARCH {
+	case "linux-amd64":
+		variant = "linux-x86_64"
+	case "linux-386":
+		variant = "linux-x86_32"
+	case "linux-arm64":
+		variant = "linux-"
+	case "darwin-amd64", "darwin-arm64":
```

Comment:
Can we add a todo comment here/in the gomod file to remind ourselves to update this when protoc supports arm-based darwin processors?